draft-ietf-acme-acme.md

```diff
@@ -3052,6 +3053,27 @@ There are certain factors that arise in operational reality that operators of
 ACME-based CAs will need to keep in mind when configuring their services.
 For example:
 
+## Key Selection
+
+ACME relies on two different classes of key pair:
+
+* Account key pairs, which are used to authenticate account holders
+* Certificate key pairs, which are used to sign and verify CSRs (and whose
+  public keys are included in certificates)
+
+Of these two types of key pair, account key pairs are far more dangerous.  While
+the compromise of a certificate key pair allows the attacker to impersonate the
+entities named in the certificate for the lifetime of the certificate, the
+compromise of an account key pair allows the attacker to take full control of
+the victim's ACME account -- potentially causing unintended issuance or
+revocation.
```

Comment:
It might be best to mention what capabilities are of concern:
1. the ability to revoke existing certificates
2. the ability to issue certificates using existing authorizations

Are these the main ones?